src/main/groovy/org/transmartproject/batch/concept/ConceptNode.groovy

```diff
@@ -35,6 +35,6 @@ class ConceptNode {
     }
 
     boolean isNew() {
-        code == null
+        i2b2RecordId == null
     }
```

Comment:
@cataphract Think about concept nodes that are higher in hierarchy than study node. e.g. `\Private Studies\`, `\Public Studies\` They are not represented in `i2b2demodata.concept_dimension` table. Hence correspondent records in `i2b2` table do not have code filled in. For such nodes the method suggested that they are new and ts-batch tried to insert them again.